packages/react/src/chip/Chip.css.ts

```diff
@@ -0,0 +1,117 @@
+import { theme } from "../styles";
+import { createVar, style } from "../vanilla-extract";
+import { type RecipeVariants, recipe } from "../vanilla-extract";
+
+const accentColorVar = createVar();
+const solidAccentColorVar = createVar();
+const subtleAccentColorVar = createVar();
+
+export const chip = recipe({
+  base: [
+    {
+      alignItems: "center",
+      display: "inline-flex",
+      flexDirection: "row",
+      gap: "4",
+      justifyContent: "center",
+      overflow: "hidden",
+      rounded: "md",
+      transition: "colors",
+    },
+    style({
+      position: "relative",
+      userSelect: "none",
+
+      selectors: {
+        "&:focus-visible": {
+          outlineOffset: "1px",
+          outlineStyle: "solid",
+          outlineWidth: "2px",
+        },
+        '&[data-state="on"]': {
+          backgroundColor: solidAccentColorVar,
+        },
+      },
+    }),
+  ],
+  variants: {
+    colorScheme: {
```

Comment:
I think we can remove colorScheme as it will be single color.

Reply:
I just wanted to wait for the final design. So kept the implementation like this. Changing this though